test/unit/profile_validation_test.rb

```diff
@@ -8,6 +8,7 @@ class ProfileValidationTest < Test::Unit::TestCase
   us_core_ig = File.join(FIXTURES_DIR, 'us_core', '*.json')
   PROFILES = {}
   Dir.glob(us_core_ig).each do |definition|
+    next if definition.end_with? '/package.json'
```

Comment:
It just polluted the logs by saying that it could not parse this file as a FHIR resource, and seeing that kind of error show up in the log made me wonder if something wrong was going on.  But I see the value in leaving it in there, because then this directory is just a copy/paste of US Core examples (I believe), without any file massaging necessary.

Reply:
Gotcha.  I was just wondering 🙂 .